src/pages/mining.md

```diff
@@ -13,10 +13,10 @@ images:
 
 ## Introduction
 
-Make sure you've followed our guide for getting a Stacks 2.0 Testnet node up and running, once completed it's only a few more steps to run a proof-of-burn miner on the testnet.
+Make sure you've followed the [Running testnet node](https://docs.blockstack.org/stacks-blockchain/running-testnet-node) tutorial. Once completed it's only a few more steps to run a proof-of-burn miner on the testnet.
```

Comment:
make sure to not do absolute urls when they are internal links. This causes a full page reload instead of a client-side navigation.